### PR TITLE
HACS fixes

### PR DESCRIPTION
**Pull request by KTibow**
Fixes as [suggested](https://github.com/hacs/default/pull/662#issuecomment-712282790) by @ludeeus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # localtuya-homeassistant
 
-A Home Assistant / Hass.io add-on for local handling of Tuya-based devices.
+A Home Assistant custom component for local handling of Tuya-based devices.
```
Comment (ludeeus) [inline README.md:5]:
```suggestion
A Home Assistant custom integration for local handling of Tuya-based devices.
```

File: hacs.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "LocalTuya",
   "domains": ["media_player", "sensor"],
```
Comment (postlund) [inline hacs.json:3]: Should be binary_sensor, cover, fan, light, sensor and switch.

Comment (KTibow) [reply hacs.json:3]: Mind using the request changes button to make it easy for me to change it?

Comment (postlund) [inline hacs.json:3]:
```suggestion
  "domains": ["binary_sensor", "cover", "fan", "light", "sensor", "switch"],
```

Comment (postlund) [reply hacs.json]: Done (had totally missed that this feature actually exists in mobile UI)

**Comment by rospogrigio** [comment]
Thank you guys, I noticed this PR just after I had pushed almost the same corrections!
Now it seems to be ready for HACS integration, all checks passed (apart from the fork thing).

**Comment by WizBangCrash** [comment]
I received this error when upgrading via HACS to 3.0.3:
```shell
2020-10-19 22:13:12 ERROR (MainThread) [custom_components.hacs.repository.integration.rospogrigio.localtuya] No file found 'manifest.json'
```

**Pull request closed without merging**